spec/influxdb/client_spec.rb

```diff
@@ -26,8 +26,8 @@
     let(:url) { subject.send(:full_url, '/') }
 
     it "GET" do
-      stub_request(:get, stub_url).to_return(body: '[]')
-      expect(subject.get(url, parse: true)).to eq []
+      stub_request(:get, stub_url).to_return(body: '{}')
+      expect(subject.get(url, parse: true)).to eq({})
```

Comment:
Maybe it is valid. This was an issue just before the PR because the multi-line parsing is merging hashes. This is still true and so a JSON response of the array form "[]" in a multi-line chunked response will probably fail, but is not required at this stage.

The fact I made the old code path continue to be used just before issuing this PR means that the array form is working, and I've reverted this change to this test.